src/gogo.py

```diff
@@ -10,6 +10,7 @@
 
 # Shortcuts may not use these names.
 RESERVED_NAMES = {'_create', '_delete', '_edit', '_list', '_ajax'}
+HTTPS_REDIRECT_URL = flask.current_app.config['HTTPS_REDIRECT_URL']
```

Comment:
@eshaverma current_app doesn't work, do you know how shall I retrieve the app config set in the Config.py?